src/user/user.service.ts

```diff
@@ -128,8 +122,8 @@ export class UserService {
 
   async uploadProfilePicture(uid: number, image: Express.Multer.File): Promise<UploadProfileResponse> {
     const { uid: foundUid } = await this.findOne({ uid: uid }, { shouldExist: true });
-    if (image.mimetype !== 'image/png' && image.mimetype !== 'image/jpeg') {
-      throw new BadRequestException('Invalid image type');
+    if (!image || (image.mimetype !== 'image/png' && image.mimetype !== 'image/jpeg')) {
+      throw new UnsupportedMediaTypeException('Invalid image type');
     }
     if (image.size > 20000000) {
```

Comment:
dont forget to remind frontend about this constraint na

Reply:
okk